diff --git a/node_modules/electron-stream/index.js b/node_modules/electron-stream/index.js
index 8524e59..5b89a67 100644
--- a/node_modules/electron-stream/index.js
+++ b/node_modules/electron-stream/index.js
@@ -79,8 +79,12 @@ Electron.prototype._onfinish = function(){
 Electron.prototype._spawn = function(url){
   debug('spawn %s', url);
 
+  var args = []
+  if (process.getuid() === 0) args.push("--no-sandbox");
+  args.push(runner);
+
   var self = this;
-  var ps = self.ps = spawn(electron, [runner], {
+  var ps = self.ps = spawn(electron, args, {
     stdio: [null, null, null, 'ipc']
   });
 
